host/class/msc/usb_host_msc/src/msc_scsi_bot.c

```diff
@@ -273,6 +291,21 @@ esp_err_t bot_execute_command(msc_device_t *device, msc_cbw_t *cbw, void *data,
     return check_csw(&csw, cbw->tag);
 }
 
+static void decode_sense_keys(cbw_sense_response_t *sense_response)
+{
+    const uint32_t key = 0x00 << 24 |
```

Comment:
Then, here you don't need to make all this shifts.

Reply:
Thx for the suggestion, I updated the `decode_sense_keys` functions with a lookup table of error codes and their descriptions. PTAL.